proposals/lip-0058.md

```diff
@@ -364,30 +364,41 @@ The value of the validators hash corresponding to the validator information and
 
 ##### Execution
 
-We use the following JSON schema for the computation of the validators hash.
+We define JSON schema for validators' BLS keys and BFT weights:
+
+```java
+validatorsBlsKeysAndBftWeightsSchema = {
```

Comment:
this is not a valid Lisk JSON schema, as the root property is not of type object. I would prefer to define the schema for the validator 

```
"type": "object",
"required": ["blsKey", "bftWeight"],
"properties": {
    "blsKey": {
        "dataType": "bytes",
        "length": BLS_PUBLIC_KEY_LENGTH,
        "fieldNumber": 1
    },
    "bftWeight": {
         "dataType": "uint64",
          "fieldNumber": 2
    }
}
```

and keep the array part in the original schema

Reply:
Agreed. It was a bit strange to me why this schema doesn't start with the type object, now I see why. Thanks!